Content.Shared/Climbing/BonkableComponent.cs

```diff
@@ -0,0 +1,42 @@
+using Content.Shared.Damage;
+using Robust.Shared.Audio;
+using Robust.Shared.GameStates;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Content.Shared.Climbing
+{
+    [RegisterComponent, NetworkedComponent]
```

Comment:
Once you the next review is done, add this attribute `[Access(typeof(SharedBonkSystem)]`